### PR TITLE
Cron fix & gitlab-ci

### PR DESCRIPTION
**Pull request by finestructure**
This should fix the issue of cron not kicking off (will confirm asap)

We're set up via Gitlab and I though it might help in general to have public images available automatically, which the added `.gitlab-ci.yml` file gives for free. I have set https://gitlab.com/finestructure/lets-nginx up to mirror https://github.com/smashwilson/lets-nginx .

This will make images available (after `docker login registry.gitlab.com`, accounts are free):

```
docker pull registry.gitlab.com/finestructure/lets-nginx:cbdcfb056b9c950d117eb02a35814d052a6844cd
```

I thought that might be useful to have.

Happy to drop that if you'd rather keep it out of Github and reduce the PR just to the cron fix. Please let me know!

### CODE REVIEW

**Comment by jbtrystram** [comment]
There was an auto build on dockerhub but it seems broken. @smashwilson do you mind looking into it ? Otherwise I am happy to set up a build on my dockerhub account..

**Comment by smashwilson** [comment]
Ahhh yup, my DockerHub account lost its connection to my GitHub account a while ago, it looks like. It's been fixed and is building now 🚀

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,11 @@
 FROM nginx
-MAINTAINER Ash Wilson <smashwilson@gmail.com>
+LABEL Ash Wilson <smashwilson@gmail.com>
```
Comment (smashwilson) [inline Dockerfile]:
```suggestion
LABEL maintainer=Ash Wilson <smashwilson@gmail.com>
```

Following the convention suggested from [the `Dockerfile` docs](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated).

It'd be nice to get other maintainers credited in here too 🤔

File: Dockerfile [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 
 #We need to install bash to easily handle arrays
 # in the entrypoint.sh script
 RUN apt-get update && apt-get install -y \
   certbot \
+  cron \
```
Comment (smashwilson) [inline Dockerfile:8]: 😆 Haha, that would help!

Comment (finestructure) [reply Dockerfile:8]: Imagine my surpise 🤣

**Comment by smashwilson** [comment]
Merged and added you as a collaborator on the repo ✨

**Comment by finestructure** [comment]
One thing to bear in mind is that I haven't confirmed the cron job is executing correctly yet. I've got a reminder set to check tomorrow morning, will report back.

**Pull request merged**